examples/leaf/wiring/example_logs/nonleaf_process_logger.goveclogger-Log.txt

```diff
@@ -0,0 +1,132 @@
+INFO Unpacking response from server
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":48}
+INFO ra: hello 3
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":49}
+INFO rb: [hi bye]
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":50}
+INFO rc: 6
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":51}
+INFO rd: 3
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":52}
+INFO re: map[hello 3:{3 hello 3 0 map[hello:{greetings mate [cool beans]}]}]
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":53}
+INFO Sending response to the client
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":54}
+INFO Unpacking go vec context from client request
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":33, "nonleaf_process_logger.goveclogger":55}
+INFO Preparing to make request for function HelloInt
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":36, "nonleaf_process_logger.goveclogger":56}
+INFO Unpacking response from server
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":36, "nonleaf_process_logger.goveclogger":57}
+INFO Preparing to make request for function HelloNothing
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":39, "nonleaf_process_logger.goveclogger":58}
+INFO Unpacking response from server
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":39, "nonleaf_process_logger.goveclogger":59}
+INFO Preparing to make request for function HelloMate
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":60}
+INFO Unpacking response from server
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":61}
+INFO ra: hello 4
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":62}
+INFO rb: [hi bye]
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":63}
+INFO rc: 8
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":64}
+INFO rd: 4
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":65}
+INFO re: map[hello 4:{4 hello 4 0 map[hello:{greetings mate [cool beans]}]}]
+nonleaf_process_logger.goveclogger {"leaf_process_logger.goveclogger":41, "nonleaf_process_logger.goveclogger":66}
+INFO Sending response to the client
```

Comment:
Remove example_logs from `examples/leaf/wiring`.  If you want to include any examples, put them in the plugin directory along with a README that explains their origin and content.